tools/debug-ui/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:19-alpine AS build
+
+WORKDIR /build
+ADD package.json /build/package.json
+ADD package-lock.json /build/package-lock.json
+RUN npm install
+
+ADD src /build/src
+ADD public /build/public
+ADD tsconfig.json /build/tsconfig.json
+RUN npm run build
+
+FROM nginx:1.21-alpine
+COPY --from=build /build/build /var/www/html
+COPY nginx.conf /etc/nginx/conf.d/default.conf
```

Comment:
Why isn't npm start (starting the local server) part of the docker image? 
Are we supposed to run this container then start it with a separate command?

Reply:
Thanks! That makes sense 

Never used nginx before and thought it was used only as a backwards proxy (caches for instance) for handling many incoming request, so wasn't aware it's also a general purpose web server.